samples/cellular/modem_shell/prj.conf

```diff
@@ -96,8 +96,8 @@ CONFIG_SMS=y
 
 # LTE link control
 CONFIG_LTE_LINK_CONTROL=y
-CONFIG_LTE_NETWORK_MODE_LTE_M_GPS=y
-CONFIG_LTE_MODE_PREFERENCE_AUTO=y
+# Use modem system mode setting by default.
+CONFIG_LTE_NETWORK_DEFAULT=y
```

Comment:
This is something that should be mentioned in release notes

Reply:
Yes, you are right. I will ask from the FT team if this change is ok for them. If it is, I'll update the changelog.

Reply:
The change shouldn't affect the FT team, because they always explicitly set the system mode. I'll update the changelog.